types.go

```diff
@@ -509,12 +509,8 @@ func sprintTxtOctet(s string) string {
 		switch {
 		case n == 0:
```

Comment:
Probably best to replace the `switch` statement here with just an `if-else` statement.

Reply:
Sure 👍